lib/generators/accessly/install/templates/db/migrate/create_permitted_action_on_objects.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePermittedActionOnObjects < ActiveRecord::Migration<%= migration_version %>
+  def change
+    create_table :accessly_permitted_action_on_objects, force: true, id: :uuid do |t|
+      t.column :segment_id, :integer, default: -1
+      t.column :action, :integer, null: false
+      t.column :actor_id, :integer, null: false
+      t.column :actor_type, :string, null: false
+      t.column :object_type, :string, null: false
+      t.column :object_id, :integer, null: false
+    end
+
+    add_index(:accessly_permitted_action_on_objects, [:segment_id, :action, :actor_id, :actor_type, :object_type, :object_id], unique: true, name: "acessly_paoo_uniq_table_idx")
```

Comment:
I'm trying to think through some scenarios that would require different indexes. I'll put some ideas in separate threads so we can talk about them separately (does that make sense or should I have just put them all in one thread?).